playpen/all_26.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/bash -e
+
+
+# You need the .bashrc that the Vagrant or dev-setup.sh environment creates. You can get it from
+# playpen/ansible/roles/dev/files/bashrc if you aren't using one of those envs.
+. ~/.bashrc
+
+# Switch back to pulp-2.6
+pstop
+
+pushd ~/devel
+for r in {pulp_deb,pulp_docker,pulp_openstack,pulp_ostree,pulp_puppet,pulp_python,pulp_rpm,pulp}; do
+    if [ -d $r ]; then
+        pushd $r
+        sudo ./pulp-dev.py -U
+        popd
+    fi
+done
+
+pushd pulp
+git checkout 2.6-testing
+find ~/devel -name "*.pyc" -delete
+sudo ./pulp-dev.py -I
+popd # pulp
+
+# This one is necessary to work around a Pulp httpd config bug, where some plugin MUST install an
+# httpd config.
+pushd pulp_rpm
+git checkout 2.6-release
```

Comment:
Elsewhere, `-dev` branches are checked out. Is there any particular reason `-release` is checked out here?